pytest.ini

```diff
@@ -16,7 +16,7 @@ env =
     AWS_REGION=ca-central-1
     ASSET_DOMAIN=assets.notification.canada.ca
     NOTIFY_EMAIL_DOMAIN=notification.canada.ca
-    SQLALCHEMY_DATABASE_URI=postgresql://postgres:postgres@localhost:5432/test_notification_api
-    SQLALCHEMY_DATABASE_READER_URI=postgresql://reader:postgres@localhost:5432/test_notification_api
+    D:SQLALCHEMY_DATABASE_URI=postgresql://postgres:postgres@localhost:5432/test_notification_api
+    D:SQLALCHEMY_DATABASE_READER_URI=postgresql://reader:postgres@localhost:5432/test_notification_api
```

Comment:
Wonderful, thanks for doing that!

In the future, could you draft up another PR to do these changes? This file and the Docker compose changes are not related to other changes.

Reply:
thanks for the note- i'll do that in the future. This was blocking me for testing the changes so I lumped it in. But in the future i'll make a separate change.